packages/manager/modules/pci/src/projects/project/storages/blocks/add/add.controller.js

```diff
@@ -177,8 +177,15 @@ export default class PciBlockStorageAddController {
   }
 
   onRegionChange() {
+    const { region } = this.storage;
     this.displaySelectedRegion = true;
     this.selectedVolumeAddon = null;
+    const volumeAddonsIds = this.volumesAvailability.plans?.filter(
+      ({ regions }) => regions.find(({ name }) => name === region.name),
+    );
+    this.consumptionVolumeAddons = this.totalConsumptionVolumeAddons.filter(
```

Comment:
no need to check nullity as you've checked it for `this.volumesAvailability.plans?`